Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -123,16 +135,21 @@ + (void)didReceiveRemoteNotification:(NSDictionary *)notification
 
 + (void)didReceiveLocalNotification:(UILocalNotification *)notification
 {
+  [[NSNotificationCenter defaultCenter] postNotificationName:RCTLocalNotificationReceived
+                                                      object:self
+                                                    userInfo:[self convertLocalNotification:notification]];
+}
+
++ (NSDictionary *)convertLocalNotification:(UILocalNotification *)notification
```

Comment:
This would be cleaner a static C function instead of a class method, since it doesn't reference self.

    static NSDictionary<NSString *, id> RCTConvertLocalNotification(UILocalNotification *notification)
    {
      ...
    }